server/app/app.go

```diff
@@ -24,7 +24,7 @@ import (
 	httpServer "github.com/kubeshop/tracetest/server/http"
 	"github.com/kubeshop/tracetest/server/http/mappings"
 	"github.com/kubeshop/tracetest/server/http/websocket"
-	linterResource "github.com/kubeshop/tracetest/server/linter/resource"
+	analyzer "github.com/kubeshop/tracetest/server/linter/analyzer"
```

Comment:
no need to use the alias if the alias has the same name as the package. You can remove them all

Reply:
Thanks! Also updating it.